python/fleetx/utils/downloader.py

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 from paddle.fluid.contrib.utils import HDFSClient, multi_download
 import time
-from env import is_first_worker
+from .env import is_first_worker
```

Comment:
Some users may want to use the model in local mode, in that we we don't know the information of first worker. Please adapt this condition for better user experience.